### PR TITLE
adds the masterIntegrationMap routeRoles

### PR DESCRIPTION
**Pull request by ankunath**
https://github.com/Shippable/api/issues/16887

### CODE REVIEW

File: migrations/v5.2.3.sql [hunk 1]
```diff
@@ -4041,6 +4041,14 @@ do $$
       roleCode := 6060
     );
 
+    -- set masterIntegrationMap routeRoles
+
+    perform set_route_role(
+      routePattern := '/masterIntegrationMap',
```
Comment (shrivara) [inline migrations/v5.2.3.sql]: Singular?

Comment (shrivara) [inline migrations/v5.2.3.sql]: This will get me all the maps right?

Comment (ankunath) [inline migrations/v5.2.3.sql]: Should it be `masterIntegrationsMap` or `masterIntegrationMaps` ?

Comment (shrivara) [inline migrations/v5.2.3.sql]: masterIntegrationMaps right? This will give me a list of maps.

Comment (ankunath) [inline migrations/v5.2.3.sql]: as discussed with @shrivara , will update to masterIntegrationCodes

**Pull request merged**